docs/source/conf.py

```diff
@@ -44,6 +44,10 @@
 # Find all the nodes and move them to the temp module folder
 node_source_files = []
 for pkg in os.listdir(os.path.abspath('../../packages/')):
+    ### !!!! TODO: REMOVE THIS AND NEXT 3 LINES AFTER THE REPOSITORY HAS BEEN CLEANED
+    if '%s.rst'%pkg not in os.listdir(os.path.abspath('packages/')):
+        print("PACKAGE %s NOT DOCUMENTED!" % pkg)
+        continue
```

Comment:
@AleksandarPetrov Do we need this?

Reply:
Yes, some of the packages had some weird structure or files that we don't use but that require libraries that are not installed which breaks the process. This check prevents attempting to build documentation for these packages because it fails if there are missing libraries. Once everything is cleaned and documented we can remove it but we're not there yet.